modules/layout/footer.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { Link } from '@oacore/design'
+
+import styles from './styles.module.css'
+
+const Footer = () => (
+  <footer className={styles.footer}>
+    <div className={styles.footerMain}>
+      <aside className={styles.footerMainLeft}>
```

Comment:
Don't include this to aside, it's not correct. Footer inside the body is a content-info, kind of aside already. It's my fault, I did not know it previously.